pkg/ui/workspaces/db-console/src/views/reports/containers/debug/index.tsx

```diff
@@ -427,7 +427,7 @@ export default function Debug() {
           />
           <DebugTableLink
             name="Heap (recent allocs)"
```

Comment:
What do you think of changing this text from `Heap (recent allocs)` to something like `All Allocs (last 5 seconds)`?

Reply:
👍🏼 SGTM thanks for clarifying